doc/api/webcrypto.md

```diff
@@ -354,12 +354,18 @@ Provides access to the `SubtleCrypto` API.
 added: v15.0.0
 -->
 
-* `typedArray` {Buffer|TypedArray|DataView|ArrayBuffer}
-* Returns: {Buffer|TypedArray|DataView|ArrayBuffer} Returns `typedArray`.
+<!--lint disable maximum-line-length remark-lint-->
+
+* `typedArray` {Buffer|Int8Array|Uint8Array|Uint8ClampedArray|Int16Array|Uint16Array|Int32Array|Uint32Array|BigInt64Array|BigUint64Array}
+* Returns: {Buffer|Int8Array|Uint8Array|Uint8ClampedArray|Int16Array|Uint16Array|Int32Array|Uint32Array|BigInt64Array|BigUint64Array} Returns `typedArray`.
```

Comment:
```suggestion
* Returns: {Buffer|Int8Array|Uint8Array|Uint8ClampedArray|Int16Array|Uint16Array|Int32Array|Uint32Array|BigInt64Array|BigUint64Array}
```

Reply:
Clarification: this "Returns `typedArray`" means "returns the passed-as-argument object back".
Are you sure it should be omitted?

Reply:
No strong opinion, but yes that would be my personal preference. That info is specified in the following paragraph.